pkg/controllers/status/cluster_status_controller.go

```diff
@@ -330,20 +330,31 @@ func getAPIEnablements(clusterClient *util.ClusterClient) ([]v1alpha1.APIEnablem
 	return apiEnablements, nil
 }
 
-func getNodeSummary(informerManager informermanager.SingleClusterInformerManager) (v1alpha1.NodeSummary, error) {
-	var nodeSummary = v1alpha1.NodeSummary{}
+func listNodesAndPods(informerManager informermanager.SingleClusterInformerManager) ([]*corev1.Node, []*corev1.Pod, error) {
```

Comment:
`ListNodes` and `ListPods` are totally independent, can we split `listNodesAndPods` to two functions?
My concern is what if we want to list another resource in the future? like `service`, then it will be a  broken change for this function, both its name and return list should be updated.

Reply:
Let's merge this before vocation and fix this then.